app/transformations/UserAnswersToETMP/API1826.scala

```diff
@@ -20,42 +20,180 @@ import play.api.libs.json._
 import transformations.Transformer
 
 object API1826 extends Transformer {
-  val transformToETMPData: Reads[JsObject] = {
-    def eventTypeNodes(events: Seq[JsObject]): JsObject = {
-      val eventDetailNodes = events.foldLeft(Json.obj())((a, b) => a ++ b)
-      if (events.isEmpty) Json.obj() else Json.obj("eventDetails" -> eventDetailNodes)
+
+  private def optField(fieldName:String, value:Option[String]) = {
```

Comment:
Might be nicer (more concise) to change this to something like: 

private def optFields(fieldNames: Seq[String], path: JsLookupResult): Seq[(String, JsString)] = 
fieldNames.flatMap { fieldName =>
    (path \ fieldName).asOpt[String].map(fieldName -> JsString(_))
}

Then you can call like this:-

JsObject(
          optFields(Seq("startDateOfOccScheme", "stopDateOfOccScheme"), occSchemeDetailsPath)
)

Just a suggestion. If you don't agree that's fine. I will approve PR anyway.